Swift Chat Demo 2/Info.plist

```diff
@@ -46,6 +46,11 @@
 	<key>NSPhotoLibraryAddUsageDescription</key>
 	<string>The app requires access the photo library for sending image message</string>
 	<key>NSMicrophoneUsageDescription</key>
-	<string>The app requires access the microphone for sending voice message</string>
+    <string>The app requires access the microphone for sending voice message</string>
+    <key>NSAppTransportSecurity</key>
+    <dict>
+        <key>NSAllowsArbitraryLoads</key>
+        <true/>
+    </dict>
```

Comment:
Please remove `NSAppTransportSecurity` setting. We should not add this to the PR.